app/src/main/java/com/tma/sparking/services/http/ParkingFieldService.java

```diff
@@ -0,0 +1,28 @@
+package com.tma.sparking.services.http;
+
+import com.google.gson.Gson;
+import com.tma.sparking.services.http.ChannelRequest;
+import com.tma.sparking.services.http.GsonParser;
+import com.tma.sparking.services.http.Parking;
+import com.tma.sparking.models.ParkingField;
+
+import java.io.IOException;
+
+import retrofit2.Call;
```

Comment:
Remove unused imports. Move this into package: com.tma.sparking.services.parkingfieldservice;